packages/lesswrong/lib/collections/posts/constants.ts

```diff
@@ -1,3 +1,7 @@
+import { DatabasePublicSetting } from "../../publicSettings";
+import QuestionAnswerIcon from '@material-ui/icons/QuestionAnswer';
+import PlayCircleOutlineIcon from '@material-ui/icons/PlayCircleOutline';
```

Comment:
Did you look into whether there was a better icon here?

Reply:
Sorry for the oversight! Added in 5c976a448747830d2641c6c7767b402ffb90ca8e.